marketplace/commercelayer-products/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2010-2019 Michel Bitter, Bleeve B.V.
```

Comment:
Why is this also Michel? I saw this name in the Commercetools app as well. We should figure out exactly which one his name belongs to.

Reply:
Updated to Contentful. This is a leftover from Cloudinary I guess which served as the base project for both... some copy pasting leftover

Reply:
So I guess we should also change it in the other LICENSE.